src/Management/ResourceBuilder.php

```diff
@@ -410,6 +412,14 @@ private function buildUser(array $data): User
         ]);
     }
 
+    private function buildOrganization(array $data): Organization
+    {
+        return $this->createObject(Organization::class, [
+            'name' => $data['name'],
+            'sys' => $this->buildSystemProperties($data['sys'])
```

Comment:
Doesn't the config require trailing commas?
Why aren't we simply using `@Symfony` as ruleset 😭